src/data/__snapshots__/transform.test.ts.snap

```diff
@@ -1335,7 +1335,7 @@ exports[`Test Transform > should transform to labelled arrows when label provide
   "fillStyle": "solid",
   "frameId": null,
   "groupIds": [],
-  "height": 130,
+  "height": 0,
```

Comment:
looks sus

Reply:
This is fine, we shouldn't be updating arrow height when bound text height exceeds hence this snapshot was updated (check textElement.ts diff)